### PR TITLE
Proposal for Group 20

### PR DESCRIPTION
**Pull request by DanStus**

### CODE REVIEW

**Comment by charlieroberts** [comment]
OK, this sounds like it could be fun! A nice way to experiment with manipulating DOM elements on the page and generating interesting user interactions. 

I am a bit worried about this "To organize items, users will be able to create nested subtasks, lists, and containers to group tasks." That sounds nice in principle, but figuring out a good UI for this could be tricky. Do you have a UI that you'd like mimic / draw inspiration from for this? Does Bootstrap give you components that might help? If not, you might want to look at other component frameworks like Semantic UI. 

tldr; find a component/css framework that does as much of the heavy lifting for you here as possible! If you find one that does a *lot* of this for you, go ahead and implement your stretch goal as an alternative challenge.
